packages/blade/tokens/global/spacing.ts

```diff
@@ -0,0 +1,29 @@
+type spacing = Readonly<{
```

Comment:
```suggestion
type Spacing = Readonly<{
```

Good to have type name in PascalCase?

```javascript
const spacing: Spacing
```